flow360/environment.py

```diff
@@ -55,6 +55,15 @@ def get_real_url(self, path: str):
     apikey_profile="default",
 )
 
+portal_api_dev = EnvironmentConfig(
+    name="portal_api_dev",
+    web_api_endpoint="https://portal-api.dev-simulation.cloud",
+    aws_region="us-east-1",
+    apikey_profile="dev",
+)
+
+FLOW360_SKIP_VERSION_CHECK = False
```

Comment:
add parameter to `EnvironmentConfig`,
`portal_web_api_endpoint = "https://..."`
Dev:
`portal_web_api_endpoint="https://portal-api.dev-simulation.cloud"`,
Uat:
`portal_web_api_endpoint="https://uat-portal-api.simulation.cloud"`,
Prod:
`portal_web_api_endpoint="https://portal-api.dev-simulation.cloud"`,

Use it:
```
url=Env.dev.get_portal_real_url(path), json=json, params=params, auth=api_key_auth
```